Userland/Services/WindowServer/Window.cpp

```diff
@@ -933,9 +929,26 @@ void Window::set_tiled(WindowTileType tile_type)
         set_maximized(false);
 
     m_tile_type = tile_type;
+    tile_type_changed(cursor_screen);
+}
+
+void Window::tile_type_changed(Screen const* cursor_screen)
+{
+    if (m_tile_type != WindowTileType::None)
+        set_rect(WindowManager::the().tiled_window_rect(*this, cursor_screen, m_tile_type));
+    else
+        set_rect(m_floating_rect);
+    send_resize_event_to_client();
+    send_move_event_to_client();
+}
 
-    set_rect(WindowManager::the().tiled_window_rect(*this, tile_type));
+void Window::send_resize_event_to_client()
+{
     Core::EventLoop::current().post_event(*this, make<ResizeEvent>(m_rect));
+}
+
+void Window::send_move_event_to_client()
+{
     Core::EventLoop::current().post_event(*this, make<MoveEvent>(m_rect));
 }
```

Comment:
Not sure if this is the related code, but when I move a window to an edge and the tiling overlay is displayed, I cannot move the mouse or window anymore. I have to get out of the "activation zone" before I see my mouse cursor move again.

Reply:
Are you sure you're not leaving the QEMU window with your mouse? This is the only way I can replicate it (if you run with `SERENITY_SCREENS=2` this doesn't happen at all because it doesn't use auto-grab focus). I'm 99% sure this is what's happening, if not can you please record a video that shows what exactly you are doing?

Reply:
And no, this code is just a minor refactor, moving this one function call into its own function.